pkg/apis/autoscaling/v1/conversion.go

```diff
@@ -362,7 +362,6 @@ func Convert_autoscaling_HorizontalPodAutoscaler_To_v1_HorizontalPodAutoscaler(i
 		}
 		// copy before mutating
 		if !copiedAnnotations {
-			copiedAnnotations = true
```

Comment:
hmm... dropping this makes it hard to reason about future changes in the method that could check this... I'd probably add a comment exempting this

Reply:
Ack. Will add lint ignore